.github/workflows/ci.yml

```diff
@@ -51,6 +51,9 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # checkout all commits, as the build result depends on `git describe` equivalent
+      # Unset the default Java in case actions/setup-java silently fails
```

Comment:
Should we report this to GH?

Reply:
Yes, we should open an issue in https://github.com/actions/setup-java but I'm not sure how to describe it.

Reply:
i guess https://github.com/trinodb/trino/issues/14141#issuecomment-1248042941 is all we know so far.